### ISSUE TITLE
Back space not readout in Turbo C Editor

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2802]
**Reported by agateinfotek on 2012-11-17 06:26**
1.In Windows XP, NVDA 2012.1. while working on Turbo C Editor, Backspace is not spoken out as well as does not work. If we Quit NVDA, Backspace works. What could be the solution for this.

**Comment by ehollig** [comment]
Closing, as the original poster did not report trying with the latest version of NVDA as @jcsteh suggested in https://github.com/nvaccess/nvda/issues/2802#issuecomment-155302016.
